Add Debezium 2.x docs (#1149)
hazelcast/hz-docs: docs/modules/pipelines/pages/cdc.adoc

If you already have Hazelcast and you skipped the above steps, make sure to
follow from here on.

> . Make sure the MySQL CDC plugin is in `lib/` directory. You need to manually download the MySQL CDC plugin from Hazelcast's Maven  https://repo1.maven.org/maven2/com/hazelcast/jet/hazelcast-enterprise-cdc-mysql/{full-version}/hazelcast-enterprise-cdc-mysql-{full-version}-jar-with-dependencies.jar[repository] and then copy it to the `lib/` directory.

Reviewer: ```suggestion
. Make sure the MySQL CDC plugin is in the `lib/` directory. You must manually download the MySQL CDC plugin from Hazelcast's Maven link:https://repo1.maven.org/maven2/com/hazelcast/jet/hazelcast-enterprise-cdc-mysql/{full-version}/hazelcast-enterprise-cdc-mysql-{full-version}-jar-with-dependencies.jar[repository, window=_blank] and then copy it to the `lib/` directory.
```

If you already have Hazelcast and you skipped the above steps, make sure to
follow from here on.

> . Make sure the MySQL CDC plugin is in the `lib/` directory. You must manually download the MySQL CDC plugin from Hazelcast's Maven link:https://repo1.maven.org/maven2/com/hazelcast/jet/hazelcast-enterprise-cdc-mysql/{full-version}/hazelcast-enterprise-cdc-mysql-{full-version}-jar-with-dependencies.jar[repository, window=_blank] and then copy it to the `lib/` directory.

Reviewer 2: ```suggestion
. Make sure the MySQL CDC plugin is in the `lib/` directory. You must manually download the MySQL CDC plugin from link:https://repo1.maven.org/maven2/com/hazelcast/jet/hazelcast-enterprise-cdc-mysql/{full-version}/hazelcast-enterprise-cdc-mysql-{full-version}-jar-with-dependencies.jar[Hazelcast's Maven repository, window=_blank] and then copy it to the `lib/` directory.
```

+
[source,bash]
----
ls lib/
----
+
You should see the following jars:
+
* hazelcast-enterprise-cdc-debezium-{full-version}-jar-with-dependencies.jar
* hazelcast-enterprise-cdc-mysql-{full-version}-jar-with-dependencies.jar
* hazelcast-enterprise-cdc-postgres-{full-version}-jar-with-dependencies.jar
+

> WARNING: If you have Hazelcast {enterprise-product-name} Edition, you need to manually download the MySQL CDC plugin from Hazelcast's Maven  https://repo1.maven.org/maven2/com/hazelcast/jet/hazelcast-jet-cdc-mysql/{full-version}/hazelcast-jet-cdc-mysql-{full-version}-jar-with-dependencies.jar[repository] and then copy it to the `lib/` directory.

Reviewer 2: ```suggestion
WARNING: If you have Hazelcast {enterprise-product-name}, you need to manually download the MySQL CDC plugin from https://repo1.maven.org/maven2/com/hazelcast/jet/hazelcast-jet-cdc-mysql/{full-version}/hazelcast-jet-cdc-mysql-{full-version}-jar-with-dependencies.jar[Hazelcast's Maven repository] and then copy it to the `lib/` directory.
```
